creature.cpp

```diff
@@ -13,16 +13,15 @@ void Creature::setLife(double life){
 	
     if(life >= 0){
         this->life = life;
-    }else{
+    } else {
         this->life = 0;
     }
 }
 
-bool Creature::isDead(){
+bool Creature::isDead() const {
 	
     if(this->getLife() > 0){
         return false;
-    }else{
-        return true;
     }
+    return true;
```

Comment:
uu, mondjuk ez menne egy sorban is szerintem :-)

Reply:
Átírva.